js/main.js

```diff
@@ -21,37 +27,37 @@ function createPokemonListElements(arr) {
   });
 }
 
-if (pokedexStorageJSON !== null) {
-  pokedexData = JSON.parse(pokedexStorageJSON);
-  createPokemonListElements(pokedexData);
+function createPokemonTypeImages(type) {
+  const pokemonTypeContainer = document.querySelector('.types-container');
+  const image = document.createElement('img');
+  image.className = 'pokemon-type';
+  image.src = 'images/' + type + '.png';
+  pokemonTypeContainer.appendChild(image);
 }
 
 const sprite = document.getElementById('sprite');
 const height = document.getElementById('height');
 const weight = document.getElementById('weight');
 const backSprite = document.getElementById('back-sprite');
-const listItems = document.querySelectorAll('li');
-listItems.forEach((value, index) => {
-  value.addEventListener('click', () => {
-    highlightEntry(value);
-    const xhr = new XMLHttpRequest();
-    xhr.open('GET', value.dataset.url);
-    xhr.responseType = 'json';
-    xhr.addEventListener('load', function () {
-      sprite.src = xhr.response.sprites.front_default;
-      backSprite.src = xhr.response.sprites.back_default;
-      height.textContent = xhr.response.height * 4 + ' in';
-      weight.textContent = xhr.response.weight * 0.2 + ' oz';
+
+function handleListItems() {
+  let listItems = document.querySelectorAll('li');
```

Comment:
This seems redundant here because you are setting `listItems` on `line 45` with the same value